runtime/test/particle-api-test.js

```diff
@@ -280,7 +284,8 @@ describe('particle-api', function() {
     recipe.normalize();
     arc.instantiate(recipe);
 
-    await util.assertViewWillChangeTo(resultsView, Result, "value", ["done", "done"]);
+    await util.assertViewWillChangeTo(resultsView, Result, "value", ["done", "done", "HELLO", "WORLD"]);
```

Comment:
i think it is better to get rid of "done" here (have the particle not put it in the results view).

Reply:
actually I think it's kinda cool that this works :)

I'm quite OK to remove it to make the test cleaner, but it's also a nice demonstration that both the transformation particle and contained particles can work on the same output views.

Reply:
i guess, you're right!
more elegant particle purpose is not as important here as demonstration of capabilities. so let's leave it as is.

thanks!

Reply:
actually, a question. when we do in a loop:

let res = resultsView.store(new resultsView.entityClass({value: 'done'}));
outView.on('change', () => {
    outView.get().then(result => {
        if (result) {
            resultsView.store(result);
        }
   })
}, this);

is it possible that the first outView change would happen before the next "done" is added? could the resultsView end up being
['done', 'HELLO', 'done', 'WORLD'] ?